selenium_tests/course_info/page_objects/course_info_detail_page_object.py

```diff
@@ -51,20 +51,13 @@ def go_to_people_page(self):
     def reset_form(self):
         self.find_element(*Locators.RESET_FORM_BUTTON).click()
 
-    def is_locator_element_present(self, locator_element):
+    def verify_buttons_to_edit_page_are_present(self):
         try:
-            self.find_element(locator_element)
+            if (self.find_element(*Locators.RESET_FORM_BUTTON).is_displayed()
+            and self.find_element(*Locators.SUBMIT_FORM_BUTTON).is_displayed()):
+                return True
         except NoSuchElementException:
             return False
```

Comment:
Don't you need another `return False` at the bottom to ensure that if either isn't displayed, but both are found, you are still returning `False`?